lte/gateway/python/magma/subscriberdb/protocols/m5g_auth_servicer.py

```diff
@@ -101,3 +112,68 @@ def M5GAuthenticationInformation(self, request, context):
                 aia, self._print_grpc_payload,
                 "M5GAuthenticationInformation Response:",
             )
+
+
+class M5GSUCIRegRpcServicer(subscriberdb_pb2_grpc.M5GSUCIRegistrationServicer):
+    """
+    gRPC based server for the AMF.
```

Comment:
This is 100% just a style preference, but usually if the comment is only one sentence long, we can omit the period. (Either way, it would be good to stay consistent)

Reply:
Done as suggested